test/multireddit/multireddit_test.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2018, the Dart Reddit API Wrapper project authors.
+// Please see the AUTHORS file for details. All rights reserved.
+// Use of this source code is governed by a BSD-style license that
+// can be found in the LICENSE file.
+
+import 'dart:async';
+import 'package:draw/draw.dart';
+import 'package:test/test.dart';
+import 'package:color/color.dart';
```

Comment:
Is this import needed? Also, sort.

Reply:
Do you mean Color? We use HexColor Constructor in the test.